ts/editable/ContentEditable.svelte

```diff
@@ -33,10 +35,18 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     const [focusHandler, setupFocusHandling] = useFocusHandler();
 
     Object.assign(api, { focusHandler });
+
+    const description = getContext<Readable<string>>(descriptionKey);
+    $: descriptionCSSValue = `"${$description}"`;
+
+    let innerHTML = "";
+    $: empty = ["", "<br>"].includes(innerHTML);
 </script>
 
 <anki-editable
+    class:empty
     contenteditable="true"
+    bind:innerHTML
```

Comment:
I'm pretty sure, this is a one-way bind: Changes to `innerHTML` within the ContentEditable component will be pushed to the element, however not the other way around.

Reply:
Thanks, I thought the `:bind` directive would always create two-way bindings. With the CSS `:empty` pseudo-class this wasn't an issue, so I forgot to test this again with the new approach.

Reply:
Is there an idiomatic way of doing this? I don't want to pollute the component with a vanilla JS two-way binding. Also, if there's an alternative to appending `<br>` to contenteditable in order to allow newlines without crashing, I would prefer the CSS (`:empty`) solution.